### PR TITLE
Enhance usage statement

### PR DESCRIPTION
**Pull request by theryangeary**
Add a more complete help output
    
    Update help output with more descriptive flags
    
    Change instances of ginh.sh to ginh
    
    Rewrap awkward formatting in readme
    
    Add Usage section to readme
    
    Add Installation section to readme

### CODE REVIEW

**Comment by theryangeary** [comment]
I tried to clean up the commit history a bit which I know is not Great and if you would rather I not I will undo it and just add my changes.

But the history does look vastly improved

File: readme.md [hunk 1]
```diff
@@ -36,9 +42,30 @@ entries=15, file=/Users/crclark/.bash_history, char==, len=78
...
+
+### From packages:
+
+#### Debian
+
+Debian packages are available [here](https://github.com/crclark96/ginh/releases).
+
+#### Arch
+
+Ginh can be installed through the
```
Comment (crclark96) [inline readme.md]: i really hate the way ginh looks capitalized

File: readme.md [hunk 2]
```diff
@@ -36,9 +42,30 @@ entries=15, file=/Users/crclark/.bash_history, char==, len=78
 -------------------------------------------------------------------------------
 ```
 
-## help
+## Installation
+
+### From source:
```
Comment (crclark96) [inline readme.md]: These headings shouldn't have trailing punctuation, according to codacy

File: readme.md [hunk 3]
```diff
@@ -7,11 +7,17 @@
...
-the `-f` flag). 
+usage: `ginh [-h] [-a] [-n entries] [-f hist_file] [-c chart_char] [-l line_len]`
+
+`ginh` generates a bar chart of your most frequently used shell commands,
+according to your shell's history file.
+
+other flags include `-n` for specifying the number of bars, `-c` to select
+the character used in each bar, `-l` to change the maximum line length
+(bar "height"), and `-a` to treat aliases as first class entries (default
+behavior is to revert aliases into their target command).
```
Comment (crclark96) [inline readme.md]: the `-f` flag description is missing now

**Review by crclark96 — CHANGES_REQUESTED** [review]
thanks for the changes, I think things might have gotten weird because of the history re-writing but I'll merge if you update these three things

**Comment by crclark96** [comment]
Thanks for rewriting this, just make sure the conflicts with master are resolved before I push the big green button

File: readme.md (hunk 3 above)
Comment (theryangeary) [reply readme.md]: I think I'm just going to replace this paragraph with the neater, more readable usage print out from the program.

**Review by crclark96 — APPROVED** [review]
Looks good

**Pull request merged**